sdk/spring/spring-cloud-azure-feature-management/README.md

```diff
@@ -192,10 +192,136 @@ feature-management:
+  | Property | Relevance | Description |
+      |----------|-----------|-------------|
+  | **Type** | Required | Must be set to `Daily`. |
+  | **Interval** | Optional | Specifies the number of days between each occurrence. Default value is 1. |
+
+- `Weekly`
+
+  The weekly recurrence pattern causes the time window to repeat on the same day or days of the week, based on the number of weeks between each set of occurrences.
+
+  | Property | Relevance | Description |
+      |----------|-----------|-------------|
+  | **Type** | Required | Must be set to `Weekly`. |
+  | **DaysOfWeek** | Required | Specifies on which day(s) of the week the event occurs. |
+  | **Interval** | Optional | Specifies the number of weeks between each set of occurrences. Default value is 1. |
+  | **FirstDayOfWeek** | Optional | Specifies which day is considered the first day of the week. Default value is `Sunday`. |
+
+  The following example will repeat the time window every other Monday and Tuesday
+
+    ```yaml
+    pattern:
+        type: "Weekly",
+        interval: 1,
+        daysOfWeek: 
+          - Monday
+          - TuesDay
+    ```
+  **Note:** `Start` must be a valid first occurrence which fits the recurrence pattern. Additionally, the duration of the time window cannot be longer than how frequently it occurs. For example, it is invalid to have a 25-hour time window recur every day.
+
+#### Recurrence Range
+
+There are three possible recurrence range type: `NoEnd`, `EndDate` and `Numbered`.
+
+- `NoEnd`
+
+  The `NoEnd` range causes the recurrence to occur indefinitely.
+
+  | Property | Relevance | Description |
+      |----------|-----------|-------------|
+  | **Type** | Required | Must be set to `NoEnd`. |
+
+- `EndDate`
+
+  The `EndDate` range causes the time window to occur on all days that fit the applicable pattern until the end date.
+
+  | Property | Relevance | Description |
+      |----------|-----------|-------------|
+  | **Type** | Required | Must be set to `EndDate`. |
+  | **EndDate** | Required | 	Specifies the date time to stop applying the pattern. Note that as long as the start time of the last occurrence falls before the end date, the end time of that occurrence is allowed to extend beyond it. |
+
+  The following example will repeat the time window every day until the last occurrence happens on April 1st, 2024.
+
+    ``` yaml
+    start: "Fri, 22 Mar 2024 18:00:00 GMT",
+    end: "Fri, 22 Mar 2024 20:00:00 GMT",
+    recurrence:
+      pattern:
+        type: "Daily",
+        interval: 1
+      range:
+        type: "EndDate",
+        endDate: "Mon, 1 Apr 2024 20:00:00 GMT"
+    ```
+
+- `Numbered`
+
+  The `Numbered` range causes the time window to occur a fixed number of times (based on the pattern).
+
+  | Property | Relevance | Description |
+      |----------|-----------|-------------|
+  | **Type** | Required | Must be set to `Numbered`. |
+  | **NumberOfOccurrences** | Required | 	Specifies the number of occurrences. |
```

Comment:
In our current logic, we not count by weeks when pattern is "weekly".
For example, we define to repeat every Monday and Tuesday from 2024/04/06. Then when we count the occurrences, the first occurrence is 2024/05/06(Monday), the second occurrence is 2024/05/07(Tuesday), the third occurrence is 2024/05/13(Monday) .... 
Each match will be counted as one occurrence.

Reply:
We should update the description to be `Specifies the number of days that it will occur.` or something similar. Also, extra space at the start of the description. and in the split between header and body.

Reply:
That sounds reasonable to me. Updated to `Specifies the number of days that it will occur.`